convert2rhel/pkghandler.py

```diff
@@ -590,3 +589,14 @@ def clear_versionlock():
         call_yum_cmd("versionlock clear", print_output=False)
     else:
         loggerinst.info("Usage of YUM/DNF versionlock plugin not detected.")
+
+
+def is_disable_and_enable_repos_has_same_repo():
+    loggerinst = logging.getLogger(__name__)
+
+    same_repos = set(tool_opts.disablerepo) & set(tool_opts.enablerepo)
+    if same_repos:
+        message = ("Convert2RHEL found identical repos in disablerepo and enablerepo options: %s\n"
+                   "Please use only in one option and run Convert2RHEL again." % same_repos)
+        print(message)
+        loggerinst.critical(message)
```

Comment:
I don't think we need to throw a critical message, we can just warn the user instead. This change also makes it a prettier print
```suggestion
        message = "Duplicate repositories was found across disablerepo and enablerepo options:"
        for repo in same_repos:
            message += "\n%s" % repo
        message += "\nThis ambiguity may have unintended consequences."
        loggerinst.warning(message)
```